feat(jsx-email): add `bgImage` and `bgColor` props to `<Column>` (#188)
shellscape/jsx-email: docs/components/column.md

## Component Props

```tsx
export interface ColumnProps extends BaseProps<'td'> {
  bgColor?: string;
  bgImage?: string;
}
```

```tsx
bgColor: string;
```

> To set background color in html email, it wraps the `bgcolor` prop of `<td>` tags

Reviewer: ```suggestion
Used to set the background color in HTML email by wrapping the `bgcolor` property of `<td>` elements
```

```tsx
bgImage: string;
```

> To set background images in html email, it wraps the `background` prop of `<td>` tags

Reviewer: ```suggestion
Used to set background images in HTML email by wrapping the `background` property of `<td>` elements
```
